### DISCUSSION TITLE
Jupyter notebook "change cell to code cell" shortcut not working

### DISCUSSION THREAD

**Discussion by kevbaer** [posit-dev/positron#4883] [Q&A] [answered]
Hi, I'm having a problem using Jupyter notebooks in positron, specifically with the shortcut Y (for code cell). When I have a new python cell, hit M to make it a markdown cell, and then hit Y to make it a code cell, the language changes to Plain Text. (See attached video) 

Let me know what I can/should do about this! Thanks!

https://github.com/user-attachments/assets/027d8183-0a14-47d9-9a73-bcd157d899b3

**Comment by timtmok** [comment, marked as answer]
This sounds like a bug. I would recommend opening an issue about this.

**Reply by kevbaer** [reply]
Opened here: https://github.com/posit-dev/positron/issues/4898

Thanks!
